packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { PlatformContext } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+import DemoMessage from 'Components/demo-message';
+import ProofOfIncomeContainer from './proof-of-income-container.jsx';
+
+const ProofOfIncome = ({ is_virtual, is_switching, refreshNotifications }) => {
+    const { is_appstore } = React.useContext(PlatformContext);
+    if (is_virtual) return <DemoMessage has_demo_icon={is_appstore} has_button={true} />;
```

Comment:
Help to fix this in your PR
```suggestion
    if (is_virtual) return <DemoMessage has_demo_icon={is_appstore} has_button />;
```

Reply:
oh, thanks. Didn’t notice it.